runtime/onert/backend/train/CMakeLists.txt

```diff
@@ -15,3 +17,24 @@ set_target_properties(${LIB_ONERT_BACKEND_TRAIN} PROPERTIES
   INSTALL_RPATH "$ORIGIN:$ORIGIN/../..")
 
 install(TARGETS ${LIB_ONERT_BACKEND_TRAIN} DESTINATION lib/nnfw/backend)
+
+if(NOT ENABLE_TEST)
+  return()
+endif(NOT ENABLE_TEST)
+
+# Unit Tests
+set(TEST_ONERT_TRAIN_BACKEND test_onert_train_backend)
+
+add_executable(${TEST_ONERT_TRAIN_BACKEND} ${TESTS})
+
+target_link_libraries(${TEST_ONERT_TRAIN_BACKEND} ${LIB_ONERT_BACKEND_TRAIN})
+# Requires linking nnfw_coverage: check header coverage
+target_link_libraries(${TEST_ONERT_TRAIN_BACKEND} nnfw_coverage)
+target_link_libraries(${TEST_ONERT_TRAIN_BACKEND} onert_core)
+target_link_libraries(${TEST_ONERT_TRAIN_BACKEND} gtest gtest_main dl ${LIB_PTHREAD})
+
+set_target_properties(${TEST_ONERT_TRAIN_BACKEND} PROPERTIES
+  INSTALL_RPATH "$ORIGIN:$ORIGIN/../lib/nnfw/backend")
```

Comment:
This line makes it possible to find train backend library, but I'm not sure if it's OK to use the relative path(`$ORIGIN/../lib/nnfw/backend`).